build/Tasks/ZipArchiveTask.cs

```diff
@@ -41,6 +41,12 @@ public override bool Execute()
                     foreach (ITaskItem fileItem in Files)
                     {
                         string destinationFileName = new FileInfo(fileItem.ItemSpec).Name;
+                        string destinationFolder = fileItem.GetMetadata("Destination");
+                        string type = fileItem.GetMetadata("Type");
+                        if (!string.IsNullOrEmpty(destinationFolder))
+                        {
+                            destinationFileName  = Path.Combine(destinationFolder, destinationFileName);
+                        }
 
                         // Prevent duplicate files from getting zipped.
                         if (uniqueFiles.Contains(destinationFileName))
```

Comment:
Perhaps this should be an error instead?

Reply:
For local builds, we get two copies of each file since one appears in the x64\ folder and another aperas in x64\InstrumentationEngine. I can address your comment in the next iteration.